### PR TITLE
Add sample ebgp connection

### PR DESCRIPTION
**Pull request by sbarguil**
#178

### CODE REVIEW

**Comment by boucadair** [comment]
Thanks @Samier.

* Can you please update the figure to show the various elements (addressing, for example) that are mentioned in the example?
* Also, please use the address ranges in [RFC](https://datatracker.ietf.org/doc/html/rfc5737).
* I don't quit understand why the two ACs are listed in the JSON.
* You may split the example in two steps: first the creation of the AC (with L2/L3 addressing), then add a new BGP session over the existing AC (peer sap/bearer-ref/neighbors).

File: draft-ietf-opsawg-teas-attachment-circuit.md [hunk 1]
```diff
@@ -1250,6 +1250,29 @@ The attachment circuit in this case use a SAP identifier to refer to the physica
 
 This scenario allows the provider to maintain a list of ACs belonging to the same customer without requiring the full service configuration.
 
+## Connect two networks through eBGP peering
```
Comment (boucadair) [inline draft-ietf-opsawg-teas-attachment-circuit.md:1253]:
```suggestion
## BGP Peering
```

File: draft-ietf-opsawg-teas-attachment-circuit.md [hunk 2]
```diff
@@ -1250,6 +1250,29 @@ The attachment circuit in this case use a SAP identifier to refer to the physica
...
+
+In this scenario, multiple networks, each represented by a Network Element (NE), connect to other networks or a Network Access Point (NAP) for interconnection purposes. Each network connects to each other through one or more Attachment Circuits (ACs).
```
Comment (boucadair) [inline draft-ietf-opsawg-teas-attachment-circuit.md:1255]: NE, NAP are not represented in the figure

File: draft-ietf-opsawg-teas-attachment-circuit.md [hunk 3]
```diff
@@ -1250,6 +1250,29 @@ The attachment circuit in this case use a SAP identifier to refer to the physica
...
+
+To facilitate routing between the interconnected networks, BGP (Border Gateway Protocol) peering sessions are established between the participating networks. BGP is used to exchange routing information and reachability announcements between those networks.
```
Comment (boucadair) [inline draft-ietf-opsawg-teas-attachment-circuit.md:1257]:
```suggestion
To facilitate forwarding between the interconnected networks, BGP peering sessions are established between the participating networks. BGP is used to exchange routing information and reachability announcements between those networks.
```

File: draft-ietf-opsawg-teas-attachment-circuit.md [hunk 4]
```diff
@@ -1250,6 +1250,29 @@ The attachment circuit in this case use a SAP identifier to refer to the physica
...
+
+~~~~ aasvg
+{::include-fold ./figures/bgp-peering-example.txt}
+~~~~
+{: #bgp-peer-network title="Illustration of Provider Network Scenario"}
+
+As depicted in the figure {{bgp-peer-network}}, each AC represents the physical or virtual link between the two networks. The AC configuration includes parameters such as interface settings, VLAN configuration (if applicable), and any additional settings required for connectivity.
```
Comment (boucadair) [inline draft-ietf-opsawg-teas-attachment-circuit.md:1264]:
```suggestion
As depicted in {{bgp-peer-network}}, each AC represents the link between the two networks. The AC configuration includes parameters such as interface settings, VLAN configuration (if applicable), and any additional settings required for connectivity.
```

File: draft-ietf-opsawg-teas-attachment-circuit.md [hunk 5]
```diff
@@ -1250,6 +1250,29 @@ The attachment circuit in this case use a SAP identifier to refer to the physica
...
+
+Once the ACs are established, BGP peering sessions are configured between the routers of the participating networks. BGP sessions are established over the Attachment Circuits, allowing the routers to exchange routing information. BGP peering configuration includes parameters such as neighbor IP addresses, BGP AS numbers, authentication settings (if required), and route filtering policies.
```
Comment (boucadair) [inline draft-ietf-opsawg-teas-attachment-circuit.md:1266]:
```suggestion
Once the ACs are established, BGP peering sessions are configured between the routers of the participating networks. BGP sessions are established over the ACs, allowing the routers to exchange routing information. BGP peering configuration includes parameters such as neighbor IP addresses, AS numbers, authentication settings (if required), etc.
```

**Review by boucadair — CHANGES_REQUESTED** [review]

**Comment by boucadair** [comment]
@Samier, please let me know when you are ready so that I can review. Thanks/

**Comment by sbarguil** [comment]
> @samier, please let me know when you are ready so that I can review. Thanks/

can you please check it now.

File: json-examples/svc/bgp-conf-peering-example.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+          "routing-protocol": [
+            {
+              "id": "BGP",
+              "type": "ietf-vpn-common:bgp-routing",
+              "bgp": {
+                "neighbor": [
+                  {
+                    "id": "NAP-1",
+                    "remote-address": "100.100.100.1",
+                    "peer-as": 65537,
```
Comment (boucadair) [inline json-examples/svc/bgp-conf-peering-example.json:16]:
```suggestion
                    "local-as": 65536,
                    "peer-as": 65537,
```

**Review by boucadair — CHANGES_REQUESTED** [review]

**Review by boucadair — APPROVED** [review]

**Pull request merged**